library-core/src/main/java/com/mikepenz/fastadapter/extensions/ExtensionsFactories.kt

```diff
@@ -17,11 +17,10 @@ object ExtensionsFactories {
     fun create(
             fastAdapter: FastAdapter<out IItem<out RecyclerView.ViewHolder>>,
             clazz: Class<out IAdapterExtension<out IItem<out RecyclerView.ViewHolder>>>
-    ): IAdapterExtension<out IItem<out RecyclerView.ViewHolder>>? {
-        if (!factories.containsKey(clazz)) {
-            return null
-        }
-        val extension = factories[clazz]
-        return extension?.create(fastAdapter)
-    }
+    ): IAdapterExtension<out IItem<out RecyclerView.ViewHolder>>? =
+            factories[clazz]?.create(fastAdapter)
```

Comment:
Shortened the logic here, but it means the same thing. There are a lot of `containsKey` usages, which may not be necessary